### ISSUE TITLE
Card: On changing the theme to High contrast black theme, analytical card as highlighted in red is not adapted to it.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3820]
Description:  On changing the theme to High contrast black theme, analytical card as highlighted in red is not adapted to it.

Expected: All UI elements shall adapt to high contrast black theme, when applied

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/98773285-9b628800-240e-11eb-8619-ad356558557a.png)

**Comment by DeepakSap14** [comment]
here chart is not fiori 3 component. It is created from third party library.

**Comment by puru-hk** [comment]
@dimamarksman I have checked the gitbook it is not mentioned chart used the third party library. Incase if you guys discussed this in the review meeting. Could you please update in Gitbook and add a comment in this issue. So that i will close this issue.

**Comment by dimamarksman** [comment]
@puru-hk 
We use third party chart cause right now we don't have our own chart in the lib . So @manjunathanagaraj asked me to add third party chart in order to mimic analytical card.

I'm not sure we should change the gitbook, cause this example description says that `Here for illustration we use google charts library to render a chart within the card content.`

@manjunathanagaraj what do you think?

**Comment by manjunathanagaraj** [comment]
@dimamarksman , While its good that we have explicitly mentioned this in our documentation, but for the sake of transparency please put this statement in tech specs so that in future we are clear on this. 
In circumstances like this, it helps QA or other team to understand better when they try to evaluate the component's behaviour.

**Comment by dimamarksman** [comment]
@puru-hk I added a note about it in the [git book](https://app.gitbook.com/@fundamental-ngx/s/docs/done/platform-card#analytical-card), please take a look.

**Comment by puru-hk** [comment]
As per the above comments we don't have the fiori3 chart in the lib. Hence we are using the third party chart. Closing this issues.
